### PR TITLE
refactor(example): refactor effects ofType

### PR DESCRIPTION
**Pull request by Fost**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## What is the new behavior?
#1461 

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/example-app/src/app/books/effects/collection.effects.ts [hunk 1]
```diff
@@ -73,5 +69,10 @@ export class CollectionEffects {
     )
   );
 
-  constructor(private actions$: Actions, private db: Database) {}
+  constructor(
+    private actions$: Actions<
```
Comment (Fost) [inline projects/example-app/src/app/books/effects/collection.effects.ts]: **lint-staged** always rewrite this block :(

how preferably to do in this situation?

**Comment by Fost** [comment]
there is the same error in the master branch

```
[error] "2018-12-10T21:12:21.286Z"  'error from lcovParse: ' 'Failed to parse string'
[error] "2018-12-10T21:12:21.289Z"  'input: ' ''
[error] "2018-12-10T21:12:21.289Z"  'error from convertLcovToCoveralls'
```

@timdeschryver how to be?

**Comment by timdeschryver** [comment]
I also noticed the `coveralls` error during a CI build but haven't had the time to take a look. All of the tests are passing, you can ignore the error.

File: projects/example-app/src/app/books/effects/collection.effects.ts (hunk 1 above)
Comment (Fost) [reply projects/example-app/src/app/books/effects/collection.effects.ts]: ok thanks

**Pull request merged**